docs/E-Land_Requirements.rst

```diff
@@ -61,17 +53,21 @@ FUN-MVS-03 - Manual setting up an energy system optimization model
 :Progress message:
 
 It is possible to add as many components as needed to the energy model that is to be simulated with the MVS.
-They can be divided into following types:
+They can be divided into following asset types:
+
+* :ref:`Energy providers <energy_providers>`
+* :ref:`Energy production <energy_production>`
+* :ref:`Energy consumption <energy_consumption>`
+* :ref:`Energy conversion <energy_conversion>`
+* :ref:`Energy storage <energy_storage>`
```

Comment:
The link for storage and consumption does not work. https://multi-vector-simulator--817.org.readthedocs.build/en/817/E-Land_Requirements.html

Reply:
Fixed with 8dc6a58be467d206f77a55cb918f9d315d7c19b3 (please check if I created duplicate) and e98697e0521a0e86a1d7f6431cf9e6a088b5109a